lib/cylc/cfgspec/suite.py

```diff
@@ -475,6 +475,22 @@ def upg(cfg, descr):
         ['visualization', 'final cycle point'],
         converter(
             lambda x: x, 'changed naming to reflect non-date-time cycling'))
+    u.deprecate(
+        '6.0.0',
+        ['scheduling', 'cycling']
+    )
+    u.deprecate(
+        '6.0.0',
+        ['scheduling', 'special tasks', 'sequential']
+    )
```

Comment:
@benfitzpatrick - can you recall why you marked "sequential" as deprecated?  It isn't cylc-5 specific, and although I don't particularly like it, it is arguably useful - simpler than putting in many explicit previous-instance triggers - sometimes.